packages/tools/lib/create-new-component/index.js

```diff
@@ -108,47 +35,102 @@ const getLibraryName = packageName => {
+
+const generateFiles = (componentName, tagName, library, packageName, isTypeScript) => {
+	componentName = capitalizeFirstLetter(componentName);
+	const filePaths = {
+		"main": isTypeScript 
+			? `./src/${componentName}.ts` 
+			: `./src/${componentName}.js`,
+		"css": `./src/themes/${componentName}.css`,
+		"template": `./src/${componentName}.hbs`,
+	};
+
+	const FileContentTemplate = isTypeScript 
+		? tsFileContentTemplate(componentName, tagName, library, packageName) 
+		: jsFileContentTemplate(componentName, tagName, library, packageName);
+
+	fs.writeFileSync(filePaths.main, FileContentTemplate, { flag: "wx+" });
+	fs.writeFileSync(filePaths.css, "", { flag: "wx+" });
+	fs.writeFileSync(filePaths.template, "<div>Hello World</div>", { flag: "wx+" });
+
+	console.log(`Successfully generated ${filePaths.main}`);
+	console.log(`Successfully generated ${filePaths.css}`);
+	console.log(`Successfully generated ${filePaths.template}`);
+
+	// Change the color of the output
+		console.warn('\x1b[33m%s\x1b[0m', `
+	Make sure to import the component in your bundle by using:
+	import ${componentName} from "./dist/${componentName}.js";`);
 }
 
-const tagName = `ui5-${camelToKebabCase(componentName)}`;
-
-const filePaths = {
-	"js": `./src/${componentName}.js`,
-	"css": `./src/themes/${componentName}.css`,
-	"hbs": `./src/${componentName}.hbs`,
-};
-const sJsFileContentTemplate = jsFileContentTemplate(componentName);
-
-fs.writeFileSync(filePaths.js, sJsFileContentTemplate, { flag: "wx+" });
-fs.writeFileSync(filePaths.css, "", { flag: "wx+" });
-fs.writeFileSync(filePaths.hbs, "<div>Hello World</div>", { flag: "wx+" });
+// Main function
+const createWebComponent = async () => {
+	const consoleArguments = process.argv.slice(2);
+	let componentName = consoleArguments[0];
+	let tagName = consoleArguments[1];
+	let language = consoleArguments[2];
+	let isTypeScript;
+
+	if (!componentName) {
+		const response = await prompts({
+			type: "text",
+			name: "componentName",
+			message: "Please enter a component name:",
+			validate: (value) => isNameValid(value),
+		});
+		componentName = response.componentName;
+	}
 
+	if (!tagName) {
+		const response = await prompts({
+			type: "text",
+			name: "tagName",
+			message: "Please enter a tag name:",
+			validate: (value) => isTagNameValid(value),
+		});
+		tagName = response.tagName;
+	}
 
-console.log(`Successfully generated ${componentName}.js`);
-console.log(`Successfully generated ${componentName}.css`);
-console.log(`Successfully generated ${componentName}.hbs`);
+	if (!language && isTagNameValid(tagName)) {
+		const response = await prompts({
+			type: "select",
+			name: "language",
+			message: "Please select a language:",
+			choices: [
+				{
+					title: "TypeScript (recommended)",
+					value: true,
+				},
+				{
+					title: "JavaScript",
+					value: false,
+				},
+			],
+		});
+		isTypeScript = response.language;
```

Comment:
`isTypeScript` is a boolean, it doesn't make sense to initialize it like this (string). Start with false, and if the language is typescript or ts, only set to true.

Reply:
hi @vladitasev ,

I think in the case of wizard ( in the prompt ) `response.language` returns a boolean value, since we are presenting the user a choice in the wizard, instead of actually typing the desired value.

As we can see from the `await prompts` if the user choose the `TypeScript (recommended)` option, that means the `value` will be assigned to `true`. 

However the `if` checks, actually checks if the argument in the **console** which is provided is either "ts" or "typescript" which respectively set the `isTypeScript` variable to `true` or `false`. Then it is of type `string` since, the user is typing it by hand.

f.e
`yarn create-ui5-element MyComponent my-component typescript`
__________________________________________________________^^^^^ the `if` checks are reffering to this arg here.

But when the user is selecting it from here (from the wizard):
```sh
? Please select a language: » - Use arrow-keys. Return to submit.
>   TypeScript (recommended)
    JavaScript
```

It is reffered as a `boolean` value.